src/acceptance-test/java/uk/gov/ida/verifyserviceprovider/builders/ComplianceToolV2InitialisationRequestBuilder.java

```diff
@@ -20,15 +20,7 @@
     private String assertionConsumerServiceUrl = "http://verify-service-provider/response";
     private String signingCertificate = TEST_RP_PUBLIC_SIGNING_CERT;
     private String encryptionCertificate = TEST_RP_PUBLIC_ENCRYPTION_CERT;
-    private V2MatchingDataset matchingDataset = new V2MatchingDataset(
-            new MatchingAttribute("Bob", true, LocalDateTime.now().minusDays(30), LocalDateTime.now()),
-            null,
-            singletonList(new MatchingAttribute("Smith", true, LocalDateTime.now().minusDays(30), LocalDateTime.now())),
-            new MatchingAttribute("NOT_SPECIFIED", true, LocalDateTime.now().minusDays(30), LocalDateTime.now()),
-            null,
-            singletonList(new MatchingAddress(true, LocalDateTime.now().minusDays(30), LocalDateTime.now(), "E1 8QS", Arrays.asList("The White Chapel Building" ,"10 Whitechapel High Street"), null, null)),
-            UUID.randomUUID().toString()
-    );
+    private MatchingDataset matchingDataset = new MatchingDataset(new MatchingAttribute("Bob", true, LocalDateTime.now().minusDays(30), LocalDateTime.now()), null, singletonList(new MatchingAttribute("Smith", true, LocalDateTime.now().minusDays(30), LocalDateTime.now())), new MatchingAttribute("NOT_SPECIFIED", true, LocalDateTime.now().minusDays(30), LocalDateTime.now()), null, singletonList(new MatchingAddress(true, LocalDateTime.now().minusDays(30), LocalDateTime.now(), "E1 8QS", Arrays.asList("The White Chapel Building" ,"10 Whitechapel High Street"), null, null)), UUID.randomUUID().toString());
```

Comment:
Is this change just from an automated formatting?

Reply:
Yeah, I was inlining it from the IDE. Looks like I forgot to fix this one after it was inlined.